src/lib.rs

```diff
@@ -558,9 +577,9 @@ impl<'a> LabelText<'a> {
     /// This includes quotes or suitable delimeters.
     pub fn to_dot_string(&self) -> String {
         match self {
-            &LabelStr(ref s) => format!("\"{}\"", LabelText::escape_default(s)),
-            &EscStr(ref s) => format!("\"{}\"", LabelText::escape_str(&s[..])),
-            &HtmlStr(ref s) => format!("<{}>", s),
+            LabelStr(ref s) => format!("\"{}\"", LabelText::escape_default(s)),
+            EscStr(ref s) => format!("\"{}\"", LabelText::escape_str(&s[..])),
+            HtmlStr(ref s) => format!("<{}>", s),
```

Comment:
Are these `ref` actually necessary? Since you removed `&` here, I don't think these are necessary anymore.